dag_status.py

```diff
@@ -0,0 +1,38 @@
+import time
+import requests
+
+def get_latest_state():
+    url = "http://localhost:8080/api/v1/dags/lakeFS_workflow/dagRuns"
+    username="admin"
+    password="admin"
+    response = requests.get( url,  auth=(username, password))
+    dag_details = {}
+    for key in response.json()['dag_runs']:
+        dag_details[key['execution_date']] = key['state']
+
+    # Creates a sorted dictionary (sorted by key)
+    from collections import OrderedDict
+
+    dict1 = OrderedDict(sorted(dag_details.items(), reverse=True))
+    state = dict1[list(dict1.keys())[0]]
```

Comment:
IIUC this is trying to get the latest `execution_date`.  It would be easier to read if we could use the Python `max` function; I believe that
```python
latest = max(response.json()['dag_runs'], key=lambda k: k['execution_date'])
state = latest['state']
```
may be shorter and easier for me to understand.

Reply:
Thanks for the suggestion. Will change it.